x/gamm/pool-models/balancer/balancer_pool.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/x/gamm/types"
 )
 
-var _ types.PoolI = &Pool{}
+var _ types.PoolExitSwapExternAmountOutExtension = &Pool{}
```

Comment:
Can we actually do a separate such interface assertion line for every expected interface. (I know that `PoolExitSwapExternAmountOutExtension` implies PoolI, but I think these are often to simplify things for the reader)